cdap-distributions/src/COPYRIGHT/junit:junit:4.13/COPYRIGHT

```diff
@@ -0,0 +1,125 @@
+Eclipse Public License - v 1.0
```

Comment:
Leave out the licenses for test dependencies. We only need what is shipped in the release.

Reply:
As discussed, not a blocker.